app/containers/Search/SearchContent/Post.tsx

```diff
@@ -63,11 +63,9 @@ const Post: React.FC<PostProps> = ({
     tags: [],
   };
 
-  if (tags === null) {
-    tags = [];
-  }
+  const tagsArray = tags === null ? [] : tags;
```

Comment:
use useState and for update use useEffect

Reply:
done: https://github.com/peeranha/peeranha-web/pull/1289